tests/spec/core/webidl-spec.js

```diff
@@ -1,15 +1,29 @@
 "use strict";
 
 import { flushIframes, makeRSDoc, makeStandardOps } from "../SpecHelper.js";
+import { IDBKeyVal } from "../../../src/core/utils.js";
+import { openDB } from "../../../node_modules/idb/build/esm/index.js";
 
 describe("Core - WebIDL", () => {
   afterAll(flushIframes);
   /** @type {Document} */
   let doc;
+  let cache;
   beforeAll(async () => {
     const ops = makeStandardOps();
     ops.config.xref = true;
     doc = await makeRSDoc(ops, "spec/core/webidl.html");
+    const idb = await openDB("xref", 1, {
```

Comment:
In future, lets export this idb cache generation from xref.js

Reply:
Agree.